### PR TITLE
Support Engineer Job posting

### PR DESCRIPTION
**Pull request by robotdan**

### CODE REVIEW

**Comment by robotdan** [comment]
Feel free to add any suggestions you may have, or we can always modify later as well.

File: site/jobs/support-engineer.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* A fantastic, customer centric attitude
+* Some programming experience
+* An aptitude for learning new technology
+* An interest in authentication and authorization technology
+
+## Technologies
+
+The requirements for this position include:
+
+* Experience in at least one programming language
```
Comment (mooreds) [inline site/jobs/support-engineer.md:50]: I would add "Experience interacting directly with customers"

**Review by mooreds — APPROVED** [review]
One suggestion, but other than that looks good.

File: site/jobs/support-engineer.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* Knowledge of OAuth2 and OpenID Connect
+
+## Beneficial Technologies
+
+* Git
+* Java
+* JavaScript or Node.js
+* HTML
+
+## Salary & Benefits
```
Comment (mooreds) [inline site/jobs/support-engineer.md:64]: Would be awesome to put the interview process in here too.

* how many interviews
* timeframe
* any takehome work?
* etc

Comment (robotdan) [reply site/jobs/support-engineer.md:64]: Added details for take-home and number of interviews. Time frame is hard, no idea when or if we will actually hire someone.  I can add something about timeframe from the start to an answer.

**Comment by robotdan** [comment]
Merging, feel free to change or suggest further changes.

**Pull request merged**